QUARKUS-5660.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-5660 Allow to create static OIDC tenants programmatically
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5660
+
+Upstream PR:
+- https://github.com/quarkusio/quarkus/pull/45294
+
+## Scope of testing
+- Test programmatically creation and configure static OIDC tenants using `OidcTenantConfig` builder
+- Test new `Oidc` interface methods - cover creation of custom OIDC tenants and default tenants with service or web application types
+
+## Getting familiar with the feature
+Get familiar with
+- https://quarkus.io/version/main/guides/security-oidc-bearer-token-authentication#programmatic-oidc-start-up
+- https://quarkus.io/version/main/guides/security-oidc-code-flow-authentication#programmatic-oidc-start-up
+- https://quarkus.io/version/main/guides/security-openid-connect-multitenancy#programmatic-startup
+
+## Existing test coverage
+Creating custom tenants using `OidcTenantConfig` builder with multiple OIDC tenant properties
```

Comment:
Both of these tests are unit tests, and so none of them verifies, that static OIDC tenants can connect to OIDC server (eg. Keycloak). Unless there are some integration tests, I am afraid we need to create them.

Reply:
Nah, the `UserInfoRequiredDetectionTest` runs against Keycloak.